src/store/api.ts

```diff
@@ -302,22 +302,22 @@ export default {
             }
             if (vehicleStorageUpdateTimeout)
                 window.clearTimeout(vehicleStorageUpdateTimeout);
-            vehicleStorageUpdateTimeout = window.setTimeout(
-                () =>
-                    set_api_storage(
-                        'vehicles',
-                        {
-                            value: state.vehicles,
-                            lastUpdate:
-                                state.lastUpdates.vehicles ??
-                                new Date().getTime(),
-                            user_id: window.user_id,
-                        },
-                        (this as unknown) as Store<RootState>,
-                        true
-                    ),
-                1000
-            );
+            // vehicleStorageUpdateTimeout = window.setTimeout(
+            //     () =>
+            //         set_api_storage(
```

Comment:
Block commentvor total remove?

Reply:
Block comment because needs to be improved.